src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -162,12 +162,10 @@ public void ValidateAppInsightsTelemetryConnectionString(RuntimeConfig runtimeCo
         ValidateConfigProperties();
         ValidatePermissionsInConfig(runtimeConfig);
 
-        // If the ConfigValidationExceptions list doesn't contain a DataApiBuilderException with connection string error message,
-        // then only we run the metadata validation.
-        if (!ConfigValidationExceptions.Any(x => x.Message.Equals(DataApiBuilderException.CONNECTION_STRING_ERROR_MESSAGE)))
-        {
-            await ValidateEntitiesMetadata(runtimeConfig, loggerFactory);
-        }
+        _logger.LogInformation("Validating entity relationships.");
```

Comment:
can this logger be in the relationship validation function?

Reply:
there are 2 relationship validation functions now, so if we don't want this message to potentially appear twice, probably better to just put it before both calls right?

Reply:
yeah this should be fine then.

 If you did put a logging message in both validation functions, they'd need to be more specific about what they're validating. e.g. validating required relationship config values are present (no mismatching counts of source/target fields etc) and then validating relationship config aligns with database schema. (backing column validation).